test/e2e/pipeline/pipeline_test.go

```diff
@@ -305,6 +305,23 @@ Waiting for logs to be available...
 		}
 	})
 
+	t.Run("Start PipelineRun with --task-run-spec", func(t *testing.T) {
```

Comment:
When I try to run e2e test, I'm seeing following log
```
package github.com/tektoncd/cli/test/e2e/pipeline: build constraints exclude all Go files in /Users/minjaekim/workspace/oss/cli/test/e2e/pipeline
```

So I've written this code, but never have run it. What should I do?